dapps/digital-twin-detail/src/components/data/DetailDataGeneral.ts

```diff
@@ -0,0 +1,43 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+// vue imports
+import Component, { mixins } from 'vue-class-component';
+
+// evan.network imports
+import { EvanComponent } from '@evan.network/ui-vue-core';
+import { EvanUIDigitalTwin } from 'dapps/digital-twin.lib';
+import { DigitalTwin } from '@evan.network/api-blockchain-core';
+
+@Component
+export default class DigitalTwinDetailDataGeneralComponent extends mixins(EvanComponent) {
+  name: string = null;
+  description: string = null;
+  type = 'TODO Type';
+  owner = 'TODO Owner';
+  twin: DigitalTwin;
+
+  async mounted() {
+    this.twin = EvanUIDigitalTwin.getDigitalTwin(this.getRuntime(), this.$route.params.address);
+    const desc = await this.twin.getDescription();
+     console.log(desc);
+    this.name = desc.name;
+    this.description = desc.description;
```

Comment:
improvement would be to already use translated descriptions with a helper function like this:

````
typescript
  _getLocalizedTemplateEntry(template: DigitalTwinTemplate, entry: string): string {
    const lang = this.$i18n.locale() || window.localStorage.getItem('evan-language');

    return template.description?.i18n[lang]?.[entry] || template.description?.[entry] || '';
  }

Reply:
This whole code will be different with Tobi's PR ;)